frontend/pages/docs/administration/schemas/create-a-schema.mdx

```diff
@@ -3,7 +3,7 @@ import DocsWrapper from 'src/docs/DocsWrapper'
 # Creating and Uploading schemas
 
 Bailo makes use of the [RJSF](https://react-jsonschema-form.readthedocs.io/en/latest/) (react-jsonschema-form) library
-in order to dynamically create its upload & deployment forms. These forms can be constructed of various existing
+in order to dynamically create its upload & access request forms. These forms can be constructed of various existing
 widgets, or even use custom widgets defined in Bailo.
```

Comment:
"These forms can be constructed of various existing widgets, or even use custom widgets defined in Bailo." -> "

These forms are constructed using existing widgets, or custom widgets that have been created specifically for Bailo."